src/OpenColorIO/fileformats/FileFormatSpi1D.cpp

```diff
@@ -121,23 +125,29 @@ CachedFileRcPtr LocalFileFormat::read(
                     ThrowErrorMessage("Only format version 1 supported", currentLine, headerLine);
                 }
             }
-            else if(StringUtils::StartsWith(headerLine, "From"))
+            else if(chunks[0] == "Version1")
+            {
+                // "Version1" is valid.
+                version = 1;
+            }
```

Comment:
I'm not sure if this is intended behavior, or the old comment just meant to note the parser would accept this, while no file should look like this. Let me know.